src/net.cpp

```diff
@@ -2202,6 +2202,9 @@ void CConnman::SetNetworkActive(bool active)
 
 CConnman::CConnman(uint64_t nSeed0In, uint64_t nSeed1In) : nSeed0(nSeed0In), nSeed1(nSeed1In)
 {
+    Options connOptions;
```

Comment:
Let's move this to the end in case anything in Init() ends up somehow relying on default values.